### ISSUE TITLE
documentation: A mandatory variable perimeter_additional_members is commented out

### ISSUE THREAD

**Issue by eeaton** [terraform-google-modules/terraform-example-foundation#936]
### TL;DR

Confusing documetnation and deployment steps: a mandatory variable perimeter_additional_members is commented out. 

- https://github.com/terraform-google-modules/terraform-example-foundation/blob/a4967445b4e4c3074d4dcd20dc8297813c3f2ece/3-networks-dual-svpc/common.auto.example.tfvars
- https://github.com/terraform-google-modules/terraform-example-foundation/blob/a4967445b4e4c3074d4dcd20dc8297813c3f2ece/3-networks-hub-and-spoke/common.auto.example.tfvars

In other stages, the optional variables are commented out and mandatory variables are not. 

There is a reference in [readme text](https://github.com/terraform-google-modules/terraform-example-foundation/tree/a4967445b4e4c3074d4dcd20dc8297813c3f2ece/3-networks-hub-and-spoke#deploying-with-cloud-build) that the variable needs to be manually replaced, but this can be easily overlooked because it's buried in directions

### Expected behavior

Variables that must be manually rewritten should be clearly called out both in text and in tfvars files, otherwise the developer will attempt to deploy before setting all variables and hit an error.

Tech editing the directions for usability (including the awareness that developers tend to skim repetitive steps, especially when the step contains a copy-pastable code snippet) would reduce friction for a developer to deploy.

### Observed behavior

Variable that must be manually rewritten is easily overlooked in both text and tfvars files. 

### Terraform Configuration

```hcl
n/a
```

### Terraform Version

```sh
n/a
```

**Comment by obriensystems** [comment]
fmichaelobrien to review - currently on 1-org before the 2 3-networks-*
